Userland/DevTools/HackStudio/Dialogs/NewProjectDialog.cpp

```diff
@@ -170,7 +170,7 @@ Optional<String> NewProjectDialog::get_available_project_name()
     return {};
 }
 
-Optional<String> NewProjectDialog::get_project_full_path()
+Optional<LexicalPath> NewProjectDialog::get_project_full_path()
```

Comment:
`LexicalPath` is meant to be used as a utility class for parsing paths, and not really as a container for passing paths around.
Let's go back to simply using `String` here.

Reply:
It is used for using ``parent()`` later, should I just construct it from a string when that's required?